### ISSUE TITLE
If/Unless modifiers

### ISSUE THREAD

**Issue by palkan** [palkan/clowne#14]
```ruby
class StageCloner < Clowne::Cloner
  # Symbol support (method is called on the clone)
  nullify :external_id, unless: :new_record?

  # Block support
  include_association :votes, if: ->(source, record, params) { params[:need_votes] }
end
```

**Comment by ssnickolay** [comment]
Is this modifier will be for all declarations (`finalize`, `nullify`, `exclude_association`...)?

**Comment by palkan** [comment]
Yep, I think it should be a part of abstract Base declaration (which we don't have now).

Frankly speaking, I'm not sure about the usefulness of this feature. Haven't had a case for it yet.

**Comment by ssnickolay** [comment]
> Frankly speaking, I'm not sure about the usefulness of this feature. Haven't had a case for it yet.

But we can add flexibility in the configuration with this enhancement) And more complexity, of course...
